### DISCUSSION TITLE
How to disable highlight window

### DISCUSSION THREAD

**Discussion by acascais** [olimorris/codecompanion.nvim#101] [Q&A] [answered]
Hi all.

First of all, my thanks to @olimorris : this plugin is awesome!

Second, a really n00b question from my side: how can i disable the diff highlight window after I prompt the model to refactor my code?
I looked at all the plugins I installed as dependencies to codecompanion, but couldn't figure out which one is responsible for the highlight window. This means that I need to quit vim to close the highlight window :x 

Thanks in advance! I'm aware the answer is probably super simple :see_no_evil: 

The highlight window I'm referring to: 
![swappy-20240821_103226](https://github.com/user-attachments/assets/70551c2b-d0d4-4424-aa46-dde8cba11ea6)

**Comment by olimorris** [comment, marked as answer]
You can disable it in the [config](https://github.com/olimorris/codecompanion.nvim/blob/f9ec8eee3f60dd41b4917716c17c9a30625ddf35/lua/codecompanion/config.lua#L544). And the [README](https://github.com/olimorris/codecompanion.nvim/tree/main?tab=readme-ov-file#inline-prompting) outlines the keymaps that exist to accept or reject the edits.

**Comment by acascais** [comment]
Oh, I thought the highlight came from one of the other plugins! That's why I didn't find the one responsible.

`ga` and `gr` do the trick. Thanks!

**Reply by olimorris** [reply]
No problem.

Odd that yours is that strange pink color. What colorscheme are you using? I thought I picked a common highlight group.
